### ISSUE TITLE
"x" close button on in-app guides do not have a "close" accessibility label

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1054]
"x" close button on in-app guides do not have a "close" accessibility label. When read on a screen reader, it simply says "tap to activate" but people will not know what tapping to activate will do. This is missing on Android.

This also means that I'm unable to close the in-app guide using voice control.

**Comment by bryce-mcmath** [comment]
Is this on notifications or on the the in-app guided tours? Both?

**Comment by knguyenBC** [comment]
My bad, in-app guides only. Not notifications in general. Fixed the title of the issue
